southbound/impl/src/main/java/org/opendaylight/ansible/southbound/AnsibleCommandServiceImpl.java

```diff
@@ -91,8 +100,24 @@ public AnsibleCommandServiceImpl(@OsgiService final DataBroker dataBroker) {
                 .setFailedEvent(failedEvent).build()).buildFuture();
     }
 
-    private Uuid runAnsibleRole(String host, String dir, String role, List<String> roleVars, List<String> ansibleVars)
-            throws ManagedProcessException {
+    public void initCommandStatus(Uuid uuid) {
```

Comment:
is anything going to be added in the future?

Reply:
yeah. this method just calls another method so wondered why even have it.

Reply:
The reason is access. The ability to update a result to any status is private, just the ability to initialize it to in progress is public for other classes to call. So that another class couldn't override the status to success if the ansible command class decided it failed